dts/msm8974/msm8974-lge-hammerhead.dts

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+#include <lk2nd.h>
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+	qcom,msm-id = <0x7e 0x96 0x20002 0x0b>;
```

Comment:
There are 8 device trees in ``boot.img ``. Not sure which one is correct.

Reply:
Does booting downstream (e.g. TWRP) work from lk2nd?

Reply:
No. And I got ``qcom,msm-id in device tree is (16) not a multiple of (12)``.
[boot_downstream.txt](https://github.com/msm8916-mainline/lk2nd/files/7149995/boot_downstream.txt)

Reply:
Ugh, well lk2nd can't deal with those strange `qcom,msm-id` :(  
They should have only 2 or 3 elements but not 4

Reply:
Sadly none of these boots. And ``[90] Device tree exact match the board: <126 150 0x20002> != <126 150 0x20002>`` looks weird.

``<0x7e 0x96>`` -> ``FAILED (remote: 'dtb not found')``

``<0x7e 0x96 0x20002>`` ->  [126_150_0x20002.txt](https://github.com/msm8916-mainline/lk2nd/files/7152918/126_150_0x20002.txt)

``<0x7e 0x96 0x20001>`` ->  [126_150_0x20001.txt](https://github.com/msm8916-mainline/lk2nd/files/7152919/126_150_0x20001.txt)

``<0x7e 0x96 0x20000>`` ->  [126_150_0x20000.txt](https://github.com/msm8916-mainline/lk2nd/files/7152920/126_150_0x20000.txt)